# HG changeset patch
# User Rik <rik@octave.org>
# Date 1544679914 28800
#      Wed Dec 12 21:45:14 2018 -0800
# Node ID f125a6ed45be0a3f57880377a4d47ea21f5c1bd8
# Parent  471513a016ec2577ac8355817d260834e56580a1
NEWS: Announce support for HiDPI/Retina screens.

* NEWS: Announce support for HiDPI/Retina screens.

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -78,16 +78,20 @@ Summary of important user-visible change
         TolFun       |     1e-6       |     1e-7
 
  ** Using "clear" with no arguments now removes only local variables
     from the current workspace.  Global variables will no longer be
     visible, but they continue to exist in the global workspace and
     possibly other workspaces such as the base workspace.
     This change was made for Matlab compatibility.
 
+ ** The Octave plotting system now supports high resolution screens,
+    i.e, those with greater than 96 DPI which are referred to as
+    HiDPI/Retina monitors.
+
  ** Figure graphic objects have a new property "Number" which is
     read-only and will return the handle (number) of the figure.
     However, if the property "IntegerHandle" has been set to "off" then
     the property will return an empty matrix ([]).
 
  ** Patch and surface graphic objects now use the "FaceNormals" property
     for flat lighting.
 
